.buildkite/shared-pipeline-vars

```diff
@@ -5,6 +5,8 @@
 
 # The ~> modifier is not currently used, but we check for it just in case
 XCODE_VERSION=$(sed -E -n 's/^(~> )?(.*)/xcode-\2/p' .xcode-version)
-export IMAGE_ID="$XCODE_VERSION"
```

Comment:
💄  Nitpick: in most other repos we insert the `xcode-` prefix during the `export IMAGE_ID=` instead of via the `sed`, i.e.:

```bash
XCODE_VERSION=$(sed -E 's/^~> ?//' .xcode-version)
…
export IMAGE_ID="xcode-$XCODE_VERSION-v4"
```

This not only makes the `sed` command simpler (and thus more readable), but it also better matches the variable names. Indeed, with that approach `XCODE_VERSION` contains just the Xcode version itself, without the `xcode-` prefix. And it's only when assigning the `IMAGE_ID` value that we add the `xcode-` prefix because that's how we name our VM images.

Reply:
Good idea 👍 Updated in https://github.com/wordpress-mobile/WordPress-iOS/commit/17b86196d1fe0b01324676f4fc86ff931d6533ed